pygount/analysis.py

```diff
@@ -706,8 +728,13 @@ def encoding_for(source_path: str, encoding: str = "automatic", fallback_encodin
     assert encoding is not None
 
     if encoding == "automatic":
-        with open(source_path, "rb") as source_file:
-            heading = source_file.read(128)
+        if file_handle is None:
+            with open(source_path, "rb") as source_file:
+                heading = source_file.read(128)
+        else:
+            file_handle.seekable and file_handle.seek(0)
```

Comment:
What for do we need to `seek(0)` here? Shouldn't the file already be at the beginning?

Reply:
Will be addressed with #106